app/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 26

```

Comment:
No reason to drop it, it's released, and allows us to use mentioned APIs without errors. targetSdkVersion is what you want to watch.